content/scalability/docs/cluster-services.md

```diff
@@ -0,0 +1,60 @@
+# Cluster Services
+
+Cluster services run inside an EKS cluster, but they are not user workloads. If you have a Linux server you often need to run services like NTP, syslog, and a container runtime to support your workloads. Cluster services are similar, supporting services that help you automate and operate your cluster. In Kubernetes these are usually run in the kube-system namespace and some are run as DaemonSets.
+
+Cluster services are expected to have a high up-time and are often critical during outages and for troubleshooting. They should run on dedicated compute instances such as a separate node group or AWS Fargate. This will ensure that the cluster services are not impacted on shared instances by workloads that may be scaling up or using more resources.
```

Comment:
I like this recommendation. When you say these services are "critical during outages" can you explain what you mean? Can they cause outages if they're not available? Are they useful for troubleshooting outages? Is their ongoing availability critical during an outage? It wasn't entirely clear to me.